### DISCUSSION TITLE
Invalid JWT when sending Anon Public Key

### DISCUSSION THREAD

**Discussion by theinfosecguy** [supabase/supabase#20170] [Questions]
I'm trying out the Supabase Edge function with Supabase Auth but getting the error `Invalid JWT` when I'm sending Anon Key from Project Settings.

Ref: https://github.com/supabase/supabase/blob/master/examples/edge-functions/supabase/functions/select-from-table-with-auth-rls/index.ts

The sample token in the code is working, but the one from the project settings is not working.

**Comment by GaryAustin1** [comment]
Are you using that code exactly with your URL and apikey changed to your anon key?
Do you know what code is getting that error?

**Comment by theinfosecguy** [comment]
Yes, that's correct.
